main/src/library/Console.flix

```diff
@@ -15,6 +15,13 @@
  */
 namespace Console {
 
+    ///
+    /// Returns true if there is a `Console` object associated with the current JVM.
+    ///
+    pub def hasConsole(): Bool & Impure =
+        import static java.lang.System.console();
```

Comment:
Add a return type and effect. (This is now soon to be required-- see other PRs).